binds/python/bind_immutable.cpp

```diff
@@ -122,6 +122,9 @@ static void bind_immutable_module(py::module &m) {
                                              return py::array(3, soma->center().data());
                                          },
             "Returns the center of gravity of the soma points")
+        .def_property_readonly("max_distance", &morphio::Soma::maxDistance,
+                               "Return the maximum distance between the center of gravity "
+                               "and any of the soma points")
```

Comment:
This makes sense for contour style somas, but for cylinder ones, it satisfies the docstring, but I'm wondering if the radius of the cylinders is accounted for.

This also applies to the centerOfGravity calculation - so maybe that should be a separate issue

Reply:
By now, it should be separate — handling should be solved for both `maxDistance` and `centreOfGravity` at the same time, /me thinks.